Sources/HipMobileUI/Views/ForgotPasswordScreenView.xaml

```diff
@@ -30,21 +30,21 @@
                            Text="{helpers:Translate ForgotPasswordScreenView_Forgot_Password}" />
                     <Label x:Name="InfoLabel"
                            HorizontalOptions="Center" VerticalOptions="Start"
-                           FontSize="Small" Text="{helpers:Translate ForgotPasswordScreenView_Info}" />
+                           FontSize="Small" Text="{helpers:Translate ForgotPasswordScreenView_Info}" Style="{DynamicResource Ssp-Regular}"/>
                     <Entry x:Name="EmailEntry" Keyboard="Email"
                            VerticalOptions="End"
                            Placeholder="{helpers:Translate ForgotPasswordScreenView_Email}"
                            Text="{Binding Email, Mode=OneWayToSource}" TextChanged="EmailEntry_TextChanged" />
                     <Label x:Name="ErrorMessageLabel"
                            HorizontalOptions="Center" VerticalOptions="Start"
-                           FontSize="Small" TextColor="Red" Text="{Binding ErrorMessage, Mode=TwoWay}" />
+                           FontSize="Small" TextColor="{StaticResource RedBarColor}" Text="{Binding ErrorMessage, Mode=TwoWay}" Style="{DynamicResource Ssp-Regular}"/>
```

Comment:
Whenever you want to use a red color for e.g. error messages, you should use the `TertiaryDarkColor` for it, not `RedBarColor`. That one is intended only for the rating page.